cdap-data-fabric/src/main/java/co/cask/cdap/data2/admin/DefaultAdmin.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app.runtime;
+package co.cask.cdap.data2.admin;
```

Comment:
I've fixed this by creating an ``AuthorizationTestModule``, that uses no-op Admin, DatasetContext and Transactional for tests that don't need these things. That class movement is not necessary now.